packages/webapp/src/App.tsx

```diff
@@ -59,62 +60,64 @@ const App = () => {
 
     return (
         <MantineProvider theme={theme}>
-            <SWRConfig
-                value={{
-                    refreshInterval: 15 * 60000,
-                    // Our server is not well configured if we enable that it will just fetch all the time
-                    revalidateIfStale: false,
-                    revalidateOnFocus: false,
-                    revalidateOnReconnect: true,
-                    fetcher,
-                    onError: (error) => {
-                        if (error.status === 401) {
-                            return signout();
+            <TooltipProvider>
+                <SWRConfig
+                    value={{
+                        refreshInterval: 15 * 60000,
+                        // Our server is not well configured if we enable that it will just fetch all the time
+                        revalidateIfStale: false,
+                        revalidateOnFocus: false,
+                        revalidateOnReconnect: true,
+                        fetcher,
+                        onError: (error) => {
+                            if (error.status === 401) {
+                                return signout();
+                            }
                         }
-                    }
-                }}
-            >
-                <SentryRoutes>
-                    <Route path="/" element={<Homepage />} />
-                    <Route element={<PrivateRoute />}>
-                        {showInteractiveDemo && (
-                            <Route path="/dev/interactive-demo" element={<PrivateRoute />}>
-                                <Route path="/dev/interactive-demo" element={<InteractiveDemo />} />
-                            </Route>
-                        )}
-                        <Route path="/:env/integrations" element={<IntegrationList />} />
-                        <Route path="/:env/integration/create" element={<CreateIntegration />} />
-                        <Route path="/:env/integration/:providerConfigKey" element={<ShowIntegration />} />
-                        <Route path="/:env/connections" element={<ConnectionList />} />
-                        <Route path="/:env/connections/create" element={<ConnectionCreate />} />
-                        <Route path="/:env/connections/create/:providerConfigKey" element={<ConnectionCreate />} />
-                        <Route path="/:env/connections/:providerConfigKey/:connectionId" element={<Connection />} />
-                        <Route path="/:env/activity" element={<Activity />} />
-                        <Route path="/:env/logs" element={<LogsSearch />} />
-                        <Route path="/:env/environment-settings" element={<EnvironmentSettings />} />
-                        <Route path="/:env/project-settings" element={<Navigate to="/environment-settings" />} />
+                    }}
+                >
```

Comment:
hard to tell what this is changing....what is the adjustment here?

Reply:
I added a wrapper for the tooltip provider. You can see those diff better with "whitespace" off
<img width="1156" alt="Screenshot 2024-05-22 at 10 32 49" src="https://github.com/NangoHQ/nango/assets/1637651/4961fa7c-8f35-4913-89cc-537d2833edd0">